climada_petals/entity/exposures/openstreetmap/osm_dataloader.py

```diff
@@ -0,0 +1,945 @@
+   'ITA' : ('europe', 'italy'),
+   'LVA' : ('europe', 'latvia'),
+   'LIE' : ('europe', 'liechtenstein'),
+   'LTU' : ('europe', 'lithuania'),
+   'LUX' : ('europe', 'luxembourg'),
+   'MKD' : ('europe', 'macedonia'),
+   'MLT' : ('europe', 'malta'),
+   'MDA' : ('europe', 'moldova'),
+   'MCO' : ('europe', 'monaco'),
+   'MNE' : ('europe', 'montenegro'),
+   'NLD' : ('europe', 'netherlands'),
+   'NOR' : ('europe', 'norway'),
+   'POL' : ('europe', 'poland'),
+   'PRT' : ('europe', 'portugal'),
+   'ROU' : ('europe', 'romania'),
+   'RUS-E' : ('europe', 'russia'), # European part of Russia
+   'SRB' : ('europe', 'serbia'),
+   'SVK' : ('europe', 'slovakia'),
+   'SVN' : ('europe', 'slovenia'),
+   'ESP' : ('europe', 'spain'),
+   'SWE' : ('europe', 'sweden'),
+   'CHE' : ('europe', 'switzerland'),
+   'TUR' : ('europe', 'turkey'),
+   'UKR' : ('europe', 'ukraine'),
+   'CAN' : ('north-america', 'canada'),
+   'GRL' : ('north-america', 'greenland'),
+   'MEX' : ('north-america', 'mexico'),
+   'USA' : ('north-america', 'us'),
+   'AUS' : ('australia-oceania', 'australia'),
+   'COK' : ('australia-oceania', 'cook-islands'),
+   'FJI' : ('australia-oceania', 'fiji'),
+   'KIR' : ('australia-oceania', 'kiribati'),
+   'MHL' : ('australia-oceania', 'marshall-islands'),
+   'FSM' : ('australia-oceania', 'micronesia'),
+   'NRU' : ('australia-oceania', 'nauru'),
+   'NCL' : ('australia-oceania', 'new-caledonia'),
+   'NZL' : ('australia-oceania', 'new-zealand'),
+   'NIU' : ('australia-oceania', 'niue'),
+   'PLW' : ('australia-oceania', 'palau'),
+   'PNG' : ('australia-oceania', 'papua-new-guinea'),
+   'WSM' : ('australia-oceania', 'samoa'),
+   'SLB' : ('australia-oceania', 'solomon-islands'),
+   'TON' : ('australia-oceania', 'tonga'),
+   'TUV' : ('australia-oceania', 'tuvalu'),
+   'VUT' : ('australia-oceania', 'vanuatu'),
+   'ARG' : ('south-america', 'argentina'),
+   'BOL' : ('south-america', 'bolivia'),
+   'BRA' : ('south-america', 'brazil'),
+   'CHL' : ('south-america', 'chile'),
+   'COL' : ('south-america', 'colombia'),
+   'ECU' : ('south-america', 'ecuador'),
+   'PRY' : ('south-america', 'paraguay'),
+   'PER' : ('south-america', 'peru'),
+   'SUR' : ('south-america', 'suriname'),
+   'URY' : ('south-america', 'uruguay'),
+   'VEN' : ('south-america', 'venezuela'),
+}
+
+"""
+nested dictionary that contains collections of relevant columns (osm_keys) and 
+key - value pairs (osm_query) to extract critical infrastructure data from an 
+osm.pbf file, via the function OSM_FileQuery().retrieve_cis()
+
+Currently implemented for:
+    * educational facilities, 
+    * electric power, 
+    * food supply, 
+    * healthcare facilities, 
+    * natural gas infrastructure,
+    * oil infrastructure, 
+    * road, 
+    * rail,  
+    * telecommunications, 
+    * water supply, 
+    * wastewater.
+
+Note: If modified, make sure that key exists in osm.config file, under the
+respective geometry/-ies. 
+"""
+DICT_CIS_OSM = {
```

Comment:
Another dictionary that could possible be saved somewhere outside the module code. But maybe ask Emanuel where best to place such dictionaries

Reply:
yes, see above.